precode.sh

```diff
@@ -6,38 +6,54 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p /task/dir1 /task/dir2 /task/dir3/dir4
```

Comment:
Папки нужно создавать относительно текущей директории, а не от корня системы
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```